### ISSUE TITLE
[graph]  GPU plugin not exported

### ISSUE THREAD

**Issue by pseudotensor** [nicolargo/glances#2542]
**Is your feature request related to a problem? Please describe.**

It's unclear why I only see certain things when using [graph].  I googled and looked through docs to see if can add/control what is dumped to svg, but I could not find anything.

E.g. on one system I see:
```
-rw-rw-r-- 1 jon jon  30636 Sep  7 22:01 processcount.svg
-rw-rw-r-- 1 jon jon  18147 Sep  7 22:01 memswap.svg
-rw-rw-r-- 1 jon jon 128786 Sep  7 22:01 percpu.svg
-rw-rw-r-- 1 jon jon  22405 Sep  7 22:01 cpu.svg
-rw-rw-r-- 1 jon jon 143222 Sep  7 22:01 fs.svg
-rw-rw-r-- 1 jon jon  26713 Sep  7 22:01 load.svg
-rw-rw-r-- 1 jon jon 344532 Sep  7 22:01 diskio.svg
-rw-rw-r-- 1 jon jon  20008 Sep  7 22:01 mem.svg
-rw-rw-r-- 1 jon jon  51417 Sep  7 22:01 network.svg
```
on another I also see:
```
-rw-rw-r-- 1 jon jon 466222 Sep  5 16:57 containers.svg
-rw------- 1 jon jon 266205 Sep  6 03:01 percpu.png
```

But I'd also like other things that glances shows, like GPU memory, usage, etc.  But none of those are dumped when using [graph] even if visible in UI.

**Describe the solution you'd like**

[graph] section should contain items one would include, and other things like GPU etc. should be dumped.

**Describe alternatives you've considered**

Could not use glances and just do it myself.

**Additional context**

None

**Comment by nicolargo** [comment]
Hi @pseudotensor 

i can not reproduce the issue on my side.

When i click on 'g', the following graphs are generated in the default /tmp folder:

```
ll /tmp/*.svg
-rw-rw-r-- 1 nicolargo nicolargo 18766 oct.   7 08:53 /tmp/containers.svg
-rw-rw-r-- 1 nicolargo nicolargo 18834 oct.   7 08:53 /tmp/cpu.svg
-rw-rw-r-- 1 nicolargo nicolargo 26324 oct.   7 08:53 /tmp/diskio.svg
-rw-rw-r-- 1 nicolargo nicolargo 19547 oct.   7 08:53 /tmp/fs.svg
-rw-rw-r-- 1 nicolargo nicolargo 19267 oct.   7 08:53 /tmp/load.svg
-rw-rw-r-- 1 nicolargo nicolargo 17629 oct.   7 08:53 /tmp/mem.svg
-rw-rw-r-- 1 nicolargo nicolargo 16669 oct.   7 08:53 /tmp/memswap.svg
-rw-rw-r-- 1 nicolargo nicolargo 40253 oct.   7 08:53 /tmp/network.svg
-rw-rw-r-- 1 nicolargo nicolargo 23524 oct.   7 08:53 /tmp/percpu.svg
-rw-rw-r-- 1 nicolargo nicolargo 20568 oct.   7 08:53 /tmp/processcount.svg

```

What's the differences between yours two systems ?

Concerning the GPU, for the moment it is not implemented, it will be in release 4.0.

**Comment by nicolargo** [comment]
In the develop branch, GPU should be exported to graph.

Can you test it ?
